boards/arm/gd32f450z_eval/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2022, Rtone.
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_INIT_PRIORITY
+	int "Board initialization priority"
+	default 50
+	help
+	  Board initialization priority.
```

Comment:
where is this used btw?

Reply:
This is indeed not required, lazy copy paste from the GD32F450I board, my bad.
I'm deleting it thx!